### PR TITLE
Include uptimeRobot script

### PR DESCRIPTION
**Pull request by john-harmony**

### CODE REVIEW

File: monitoring/uptimerobot_create.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Latest distribution_config.txt can be found at
+# https://s3.console.aws.amazon.com/s3/object/harmony-benchmark/logs/2019/06/24/001727/distribution_config.txt?region=us-west-1&tab=overview
+# 
+
+create_monitors () {
+   cat distribution_config.txt | awk '{print $1,$4}' >  genesis_shard_ip.txt
+   while read ip shard
+   do  
+   curl -X POST -H "Cache-Control: no-cache" -H "Content-Type: application/x-www-form-urlencoded" -d 'api_key='${api_key}'&format=json&type=4&sub_type=99&url='${ip}'&port=6000&friendly_name='${shard}-${ip}'&alert_contacts=0754344_0_0-2840345_0_0' "https://api.uptimerobot.com/v2/newMonitor"
```
Comment (LeoHChen) [inline monitoring/uptimerobot_create.sh]: alert_contacts should be parameterized.

File: monitoring/uptimerobot_create.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Latest distribution_config.txt can be found at
+# https://s3.console.aws.amazon.com/s3/object/harmony-benchmark/logs/2019/06/24/001727/distribution_config.txt?region=us-west-1&tab=overview
+# 
+
+create_monitors () {
+   cat distribution_config.txt | awk '{print $1,$4}' >  genesis_shard_ip.txt
```
Comment (LeoHChen) [inline monitoring/uptimerobot_create.sh]: in the same aws folder, there is a hosts.txt contains all the ip address already.

File: monitoring/uptimerobot_create.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   while read ip shard
+   do  
+   curl -X POST -H "Cache-Control: no-cache" -H "Content-Type: application/x-www-form-urlencoded" -d 'api_key='${api_key}'&format=json&type=4&sub_type=99&url='${ip}'&port=6000&friendly_name='${shard}-${ip}'&alert_contacts=0754344_0_0-2840345_0_0' "https://api.uptimerobot.com/v2/newMonitor"
+ done < genesis_shard_ip_full.txt
```
Comment (LeoHChen) [inline monitoring/uptimerobot_create.sh]: why different file name? please put the file name into a parameter.

File: monitoring/uptimerobot_create.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+}
+
+#####Main#####
+
+api_key=`cat uptimerobot_api_key.txt`
+cat distribution_config.txt | awk '{print $1,$4}' >  genesis_shard_ip.txt
```
Comment (LeoHChen) [inline monitoring/uptimerobot_create.sh]: redundant file as hosts.txt.

File: monitoring/uptimerobot_create.sh [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Latest distribution_config.txt can be found at
```
Comment (LeoHChen) [inline monitoring/uptimerobot_create.sh]: no usage, no shebang.

File: monitoring/uptimerobot_create.sh (hunk 2 above)
Comment (alajko) [reply monitoring/uptimerobot_create.sh]: Lets use hosts.txt @john-harmony let me know if you need help getting it.

File: monitoring/uptimerobot_create.sh (hunk 5 above)
Comment (alajko) [reply monitoring/uptimerobot_create.sh]: +1

**Comment by alajko** [comment]
@john-harmony high level comment is, we should parameterize all file input and have a usage function. A good script to look here is `https://github.com/harmony-one/harmony/blob/master/scripts/wallet.sh `or even mystical `mystatus.sh`

**Pull request merged**